### PR TITLE
[tests-only][full-ci] adding test for editing space with role space editor without version

### PR DESCRIPTION
**Pull request by ghost**
## Description

This PR extends test for role `Space editor without version`
```feature
Scenario: space member edits the space with role 'Space Editor Without Versions'
```

## Related Issue

- Related issue: https://github.com/owncloud/ocis/issues/10090

## How Has This Been Tested?

- test environment:
- test case 1:
- test case 2:
- ...

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [ ] Tests only (no source changes)

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link>

### CODE REVIEW

File: tests/acceptance/features/apiSpaces/changeSpaces.feature [hunk 1]
```diff
@@ -577,3 +577,23 @@ Feature: Change data of space
     Then the HTTP status code should be "404"
     When user "Carol" sends PATCH request to the space "Project Jupiter" of user "Alice" with data "{}"
     Then the HTTP status code should be "404"
+
+  @env-config
+  Scenario: space member edits the space with role 'Space Editor Without Versions'
```
Comment (prashant-gurung899) [inline tests/acceptance/features/apiSpaces/changeSpaces.feature:582]:
```suggestion
  Scenario: space member with role 'Space Editor Without Versions' edits the space 
```

File: tests/acceptance/features/apiSpaces/changeSpaces.feature [hunk 2]
```diff
@@ -577,3 +577,23 @@ Feature: Change data of space
...
+  Scenario: space member with role 'Space Editor Without Versions' edits the space
+    Given the administrator has enabled the permissions role "Space Editor Without Versions"
+    And these users have been created with default attributes and without skeleton files:
+      | username |
+      | Carol    |
+    And user "Alice" has sent the following space share invitation:
+      | space           | Project Jupiter               |
+      | sharee          | Carol                         |
+      | shareType       | user                          |
+      | permissionsRole | Space Editor Without Versions |
```
Comment (amrita-shrestha) [inline tests/acceptance/features/apiSpaces/changeSpaces.feature:591]: do we have a similar test for  | permissionsRole | Space Editor |. How about testing both on same scenario using outline

Comment (saw-jan) [reply tests/acceptance/features/apiSpaces/changeSpaces.feature:591]: step `the administrator has enabled the permissions role "<role>"` should allow other roles as well.

**Review by saw-jan — APPROVED** [review]
:+1:

**Pull request merged**